packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -150,102 +135,11 @@ export class StripeHandler {
-    let result: ClientIdCapabilityMap = {};
-
-    const planConfigs = await this.stripeHelper.allMergedPlanConfigs();
-    const capabilitiesForAll: string[] = [];
-    for (const plan of await this.stripeHelper.allAbbrevPlans()) {
-      const metadata = metadataFromPlan(plan);
-      const pConfig = planConfigs?.[plan.plan_id] || {};
-
-      capabilitiesForAll.push(
-        ...commaSeparatedListToArray(metadata.capabilities || ''),
-        ...(pConfig.capabilities?.[ALL_RPS_CAPABILITIES_KEY] || [])
-      );
-
-      result = ClientIdCapabilityMap.merge(
-        result,
-        clientIdCapabilityMapFromMetadata(metadata || {}, 'capabilities:')
-      );
-
-      if (pConfig.capabilities) {
-        Object.keys(pConfig.capabilities)
-          .filter((x) => x !== ALL_RPS_CAPABILITIES_KEY)
-          .forEach(
-            (clientId) =>
-              (result[clientId] = (result[clientId] || []).concat(
-                pConfig.capabilities?.[clientId]
-              ))
-          );
-      }
-    }
-
-    return Object.entries(result).map(([clientId, capabilities]) => {
-      // Merge dupes with Set
-      const capabilitySet = new Set([...capabilitiesForAll, ...capabilities]);
-      return {
-        clientId,
-        capabilities: [...capabilitySet],
-      };
-    });
-  }
-
-  /**
-   * Only log every $sampleRate events
-   */
-  private logToSentry(eventId: string, sampleRate = 100) {
-    const counter = this.sentryLogCounter.get(eventId) || 0;
-    this.sentryLogCounter.set(eventId, counter + 1);
-    // Use counter without increment, so that first Sentry event is logged
-    return !(counter % sampleRate);
-  }
-
   async getClients(request: AuthRequest) {
     this.log.begin('subscriptions.getClients', request);
 
-    const clientsFromStripe = await this.getClientsFromStripe();
-
-    if (!this.capabilityManager) {
-      if (this.logToSentry('getClients.CapabilityManagerNotFound')) {
-        Sentry.withScope((scope) => {
-          scope.setContext('getClients', {
-            msg: `CapabilityManager not found.`,
-          });
-          Sentry.captureMessage(
-            `CapabilityManager not found.`,
-            'error' as SeverityLevel
-          );
-        });
-      }
-
-      return clientsFromStripe;
-    }
-
     try {
-      const clientsFromContentful = await this.capabilityManager.getClients();
-
-      if (_.isEqual(clientsFromContentful, clientsFromStripe)) {
-        return clientsFromContentful;
-      }
-
-      if (this.logToSentry('getClients.NoMatch')) {
-        Sentry.withScope((scope) => {
-          scope.setContext('getClients', {
-            contentful: clientsFromContentful,
-            stripe: clientsFromStripe,
-          });
-          Sentry.captureMessage(
-            `Returned Stripe as clients did not match.`,
-            'error' as SeverityLevel
-          );
-        });
-      }
-
-      return clientsFromStripe;
+      return this.capabilityService.getClients();
```

Comment:
For this `try...catch` to catch the error from `this.capabilityService.getClients()`, you'll have to add an `await` at the start, else it'll return before the error is thrown.

That said, do we need the `try...catch` here? Could we just let hapi handle the error. Do the other handlers have a catch?